src/pages/AddItem.js

```diff
@@ -4,6 +4,7 @@ import { useCollection } from 'react-firebase-hooks/firestore';
 import Swal from 'sweetalert2';
 import { useHistory } from 'react-router-dom';
 import Button from '../components/Button';
+import { isPatternLike, genericTypeAnnotation } from '@babel/types';
```

Comment:
As suspected it doesn't appear to do anything and likely one of those funny editor (i use VS code and it does it all the time) things.

Reply:
Cool. I'll remove it.

Reply:
Removed!